pkg/apis/api.acorn.io/v1/types.go

```diff
@@ -424,7 +424,7 @@ type Config struct {
 	AWSIdentityProviderARN         *string         `json:"awsIdentityProviderArn" name:"aws-identity-provider-arn" usage:"ARN of cluster's OpenID Connect provider registered in AWS"`
 	EventTTL                       *string         `json:"eventTTL" name:"event-ttl" usage:"Amount of time an Acorn event will be stored before being deleted (default '168h' - 7 days)"`
 	Features                       map[string]bool `json:"features" name:"features" boolmap:"true" usage:"Enable or disable features. (example foo=true,bar=false)"`
-	CertManagerIssuer              *string         `json:"certManagerIssuer" name:"cert-manager-issuer" usage:"The name of the cert-manager cluster issuer to use for TLS certificates on custom domains (default letsencrypt-prod)" default:"letsencrypt-prod"`
+	CertManagerIssuer              *string         `json:"certManagerIssuer" name:"cert-manager-issuer" usage:"The name of the cert-manager cluster issuer to use for TLS certificates on custom domains" default:""`
```

Comment:
I'm actually wondering, if we could create some sort of "acorn-protected" issuer.
I.e. you don't use cluster-issuer, but a namespaced issuer that is installed into the acorn-system namespace.
Challenges will happen completely in acorn-system namespace and only the resulting TLS secret is copied to the app namespace :thinking: 
Might be over-engineered :sweat_smile:

Reply:
Yeah using issuer doesn't make any sense unless we do some magic like copying certs around like you said

Reply:
I think we actually do that already here: https://github.com/acorn-io/acorn/blob/main/pkg/publish/cert.go#L131-L142